### PR TITLE
New package: musescore-2.0.2

### PR DESCRIPTION
**Pull request by jarbowski**

### CODE REVIEW

File: srcpkgs/musescore/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'musescore'
+pkgname=musescore
+version=2.0.2
+revision=1
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/musescore/template]: Why?

**Comment by jarbowski** [comment]
because the qt5-things in makedepends are only x86
seems logic to me but I may be wrong
built and works fine on my 64bit box
don't know why travis stopped building it

**Comment by Vaelatern** [comment]
Leave off the `only_for_archs` line please.

**Comment by jarbowski** [comment]
@Vaelatern
ok, so what could I do now?
what was the purpose of removing the only_for_archs line?
it seems that it only failed to build with musl
and can I push again after these qt5-things recent rebuilds?

**Comment by Vaelatern** [comment]
```
case "${XBPS_TARGET_MACHINE}" in
        *-musl) broken="https://s3.amazonaws.com/archive.travis-ci.org/jobs/119317117/log.txt";;
esac
```

**Comment by Vaelatern** [comment]
Please fix the xlint errors Travis highlighted.

**Comment by 38github** [comment]
I look forward to have this installed in Void.

**Comment by Vaelatern** [comment]
Seems it's a proper build error.  Try `disable_parallel_build=disable` to test if that fixes the issues.

**Comment by jarbowski** [comment]
I close the PR because a new version is available. I will try disable_parallel_build=disable if it still fails to build.

**Pull request closed without merging**